pkg/asset/store/store.go

```diff
@@ -335,13 +336,16 @@ func (s *storeImpl) load(a asset.Asset, indent string) (*assetState, error) {
 // purge deletes the on-disk assets that are consumed already.
 // E.g., install-config.yaml will be deleted after fetching 'manifests'.
 // The target asset is excluded.
-func (s *storeImpl) purge(excluded asset.WritableAsset) error {
+func (s *storeImpl) purge(excluded []asset.WritableAsset) error {
+STORE_ASSETS_LOOP:
 	for _, assetState := range s.assets {
 		if !assetState.presentOnDisk {
 			continue
 		}
-		if reflect.TypeOf(assetState.asset) == reflect.TypeOf(excluded) {
-			continue
+		for _, a := range excluded {
+			if reflect.TypeOf(assetState.asset) == reflect.TypeOf(a) {
+				continue STORE_ASSETS_LOOP
```

Comment:
Hmm.. can we avoid the jump with map or internal func ?

Reply:
Fixed with c9ccee9be...d3d8cade3.